src/adapter/breakpoints.ts

```diff
@@ -231,6 +262,20 @@ export class Breakpoint {
     this._setUrlLocations.clear();
     await Promise.all(promises);
   }
+
+  /**
+   * Compares this breakpoint with the other. String comparison-style return:
+   *  - a negative number if this breakpoint is before the other one
+   *  - zero if they're the same location
+   *  - a positive number if this breakpoint is after the other one
+   */
+  public compare(other: Breakpoint) {
```

Comment:
Not obvious that it's only valid for breakpoints on the same Source, maybe it should assert that